### PR TITLE
(WIP) Fixes Some Preferences Typos

### LINKED ISSUE #3163: Viewer Shrink Setting Typo (one extra space)
The setting, in Preferences > Interface, has two spaces between Viewer and Shrink rather than one.

### PR DESCRIPTION
**Pull request by MrBlockCat**
Fixes #3163, but also:

Changes bottom restart text from "* Changes will take effect the next time you run Toonz" to "* Changes will take effect the next time you run OpenToonz"

Makes spacing consistent for the '*'s and ':'s

Added a space between a word and a '('

Changed a spelling of XSheet to Xsheet

Changed "Bg" to "BG" to be consistent with others in Colors tab

And more!

### CODE REVIEW

**Comment by MrBlockCat** [comment]
I just noticed because of how translations work I may need to tweak those as well, so I've marked this as WIP for now in case that is the case

File: toonz/sources/translations/spanish/toonz.ts [hunk 1]
```diff
@@ -7063,7 +7059,7 @@ También establecer la ruta de salida a esa carpeta.</translation>
         <translation type="vanished">   Color de fotogramas anteriores: </translation>
     </message>
     <message>
-        <source>Display Lines Only </source>
+        <source>Display Lines Only</source>
```
Comment (manongjohn) [inline toonz/sources/translations/spanish/toonz.ts]: You do not correct `source` typos here. You need to correct them in code.  Text is made translatable in code with the `tr()` function.

There is a script that is run which updates the `source` data in these .ts files. Afterwards, a member of the community who speaks that language natively will correct the `translation` data. If you modify it here but not in code, the changes will get lost when we run the script which usually happens just before a release.

Comment (manongjohn) [reply toonz/sources/translations/spanish/toonz.ts]: Also, after the .ts files are updated, you need to generate the .qm files and place them in the stuff/config/loc language folder.

**Comment by manongjohn** [comment]
> over 1,000 warnings and 19 errors

You can ignore the warnings. What are some of the errors?

**Comment by MrBlockCat** [comment]
Screen Shot 2020-03-18 at 1 11 19 AM
This is all of the errors, I think. I'm trying to build in Xcode, following the directions opentoonz has... I had an error trying to run it in Terminal following the directions, so this was the alternative, but the Terminal had gotten far enough that it had written some make files I'd had to delete. Because of that, I think there's a decent chance it did something else I didn't catch which is why I'm likely to start over. What's weird, then, though, is the error in Terminal...

**Comment by manongjohn** [comment]
I had to fix the Travis build scripts to correct this issue for macOS builds. I guess the macOS build instructions need to be updated to include this step as well.

Verify this symbolic file exists:  ls -l /usr/local/lib/cmake/glew
If so: rm /usr/local/lib/cmake/glew

**Comment by MrBlockCat** [comment]
I can't see your two comments about the translation files for some reason, but fortunately I got am email saying them. Does what you said mean I don't have to touch the translation files and should revert the commits here that do, then?

**Comment by MrBlockCat** [comment]
Also, I'll try redoing the OpenToonz build with the new commands!

**Comment by manongjohn** [comment]
Sorry. I deleted them in part because when I got the notification of the commit, I only saw the 2nd commit where you were updating the .ts file. I didn't see the code changes you made until after I posted it.

As for updating the .ts files. 

1. If you do edit each file to match the code, when we run the script close to next release to update the files, it should not see it as a new string to translate. This is more work for you.

2. If you don't edit each one manually to match the code, when we run the scripts it may see the changes you made as completely new entries that need to be translated. The people who do translations will need to re-translate them. The old one will likely appear as "vanished" so hopefully they can take the old translations and copy to new one, but may not keep whatever changes you made. Ultimately, this is more work for translators.

Either way, you'd also need to create updated .qm files otherwise they won't see the corrected translation on nightly builds and may suddenly stuff that used to be translated is no longer translated.

**Comment by MrBlockCat** [comment]
After deleting the opentoonz folder and starting anew from there, I realized I messed up last time on putting the QT lib directory into CMakeLists.txt. This time it seems to have built successfully in Xcode! Interestingly, there never was a /usr/local/lib/cmake/glew folder to remove for me, but if it shows up sometimes it's good to be sure to remove it!

**Comment by MrBlockCat** [comment]
There were these two errors, the highlighted one of which the docs don't say should be ignored, but I guess I'll see what happens: At least Xcode said it successfully built this time!

Screen Shot 2020-03-18 at 2 21 00 PM

**Comment by manongjohn** [comment]
I noticed that the last time I had to rebuild a macOS build environment. You can ignore that one.

**Comment by MrBlockCat** [comment]
Yup, it worked and I'm in OpenToonz through the build! I'm having a slight oddity where it's somehow using a vector toolbar arrow that's gray rather than the normal black raster one. I'm not even sure where it's getting that icon from, since the one I made definitely looks different, and I built the main opentoonz/opentoonz branch which should mean none of those icons are in there (none of the others are). The normal arrow is still in the right place in the build's code... Ah well, that's a seperate issue I guess. I'll try and rebuild with some of my branches now, thanks for the help!

**Comment by MrBlockCat** [comment]
This should be close to done now! For some reason, when I build this branch, it shows the updated text in English, but when I switch to another language the text that was updated stays in English, even if it previously wasn't. Am I missing something besides updating the .ts and .qm files I did?

**Comment by MrBlockCat** [comment]
Ah hah! I figured out the problem. Since the .qm files are stored in the stuff folder, they didn't get updated automatically when I ran the new version of OpenToonz. I'm not sure why the .qm files are stored in the stuff folder, but I guess that goes back to that issue that's already open where the stuff folder should be reorganized. In the meantime, now that I've updated my stuff folder this seems to work great!

**Comment by MrBlockCat** [comment]
I've noticed another inconsistency in the Preferences window which I'll fix here before I think this PR is ready to merge (assuming no other issues are found by anyone else), but now that I've figured out the stuff folder issue that means I'm much closer to done than I'd expected!

**Comment by MrBlockCat** [comment]
Everything should now be set, with no regressions compared to without the PR in settings translations... With one exception. I was manually going through the settings with each language and I noticed when it's set to Korean, "Undo Memory Size (MB):" now doesn't translate. I've tried to figure out why this happens, but it seems to me like the code is identical to languages that do translate it. It's very odd!

**Comment by MrBlockCat** [comment]
...Yeah, I thought the typo fix PR might cause these conflicts. Let me fix those so the only problem is the Korean translation again...

**Comment by MrBlockCat** [comment]
Unfortunately, I'm not good enough at Git (yet?) to figure out how to fix these conflicts, so I'm going to take this PR down and make another with an updated repository. Sorry!

**Comment by gab3d** [comment]
@MrBlockCat 
I translate for Spanish.
Sorry to come a little late on this, but I'm playing catch-up, since I've had to be absent for a month or so...

Regarding the changes to typos and translations, I'd prefer to see the strings changed as Vanished, when I open the .ts files for translation again, so to be able to modify those myself.

I saw you've changed some of the extra spacings I've inserted between words (obviously thinking there were mistakenly there), but those were put on purpose to help with the layout of the texts inside the Preferences panels. 😉

**Comment by MrBlockCat** [comment]
@gab3d Ah, dang. I’ll try and keep the vanished even for typos in mind in the future, sorry! For this PR specifically, which spaces were intentional? And are they only ones in Spanish that are intentional, or should ones in English or other languages also be reverted? I can at least fix the spacing if you’d like...

**Comment by gab3d** [comment]
@MrBlockCat no problem.
basically, when you see two words separated by more than one space... well, it was intentional. 😄

**Comment by MrBlockCat** [comment]
@gab3d Okay! And that's in all languages, and doesn't include spaces before or after all the words?

**Comment by gab3d** [comment]
@MrBlockCat 
nope, I'm talking about Spanish only.
I don't know what other translators did or not, really.

**Comment by MrBlockCat** [comment]
Phew, that's a relief. If English was affected I'd also have to update all the language files, so I'm really glad it's not! Thanks for letting me know about the spacing thing though! I'll try and get a PR fixing that within a week, hopefully sooner.

**Comment by gab3d** [comment]
@MrBlockCat 
thank you! :)

**Comment by MrBlockCat** [comment]
@gab3d Sorry, but as I was changing some other new wording things that were noticed, I realized I don't fully understand what you'd like to see for vanishing- Would you like me to change the old spelling to vanished, but also add the new spelling with the old spelling's translation to go with it? Or would you rather I made the new and old spellings both marked as vanished? Or something else? Thank you!

**Comment by gab3d** [comment]
@MrBlockCat
You can just make any changes you see appropriate to the English strings, marking the translations as Vanished, so when I see the .ts files I can check to see what has changed and give them the correspondent translations fixes.
That's it...

**Pull request closed without merging**